benches/Cargo.toml

```diff
@@ -4,5 +4,18 @@ version = "0.1.0"
 edition = "2018"
 
 [dependencies]
-harfbuzz_rs = "1.1.2"
+harfbuzz_rs = {git = "https://github.com/harfbuzz/harfbuzz_rs/", rev = "43f0fb5"}
 rustybuzz = { path = "../" }
+
+[features]
```

Comment:
We need just `hb` feature. All languages should be enabled by default. `cargo bench` can filter benches if needed.
And macos should be always enabled, just behind a `cfg`.

Reply:
Oh! Didn't know that this could be filtered.